src/util/common-util.ts

```diff
@@ -11,7 +11,12 @@ import moment from 'moment'
 export default class CommonUtils {
   // Formatting date in the desired format for displaying in the template
   static formatDisplayDate (date: Date, format?: string) {
-    return date ? moment(date).format(format || 'MMM DD, YYYY') : ''
+    if (date) {
+      const moment = require('moment-timezone')
```

Comment:
can't you just import moment-timezone at the top?

Reply:
Also I don't think this is necessary? 

![image](https://user-images.githubusercontent.com/3484109/205123888-cfebdb34-e306-4b28-afe3-aeb78068f30f.png)